ci-setups/mr-title-validation.md

```diff
@@ -0,0 +1,90 @@
+
+# Merge Request Validation CI/CD Setup on Gitlab
```

Comment:
`Merge Request Validation CI/CD Setup on Gitlab` -> `Merge Request Title Validation CI/CD Setup on GitLab`

Reply:
Done, made the change.